gazebo_ros/src/gazebo_ros_api_plugin.cpp

```diff
@@ -36,6 +36,7 @@ GazeboRosApiPlugin::GazeboRosApiPlugin() :
   pub_model_states_connection_count_(0),
   pub_clock_frequency_(0)
 {
+  robot_namespace_.clear();
```

Comment:
I agree, but I just put back the code that was already there, if you check the overall diff you shouldn't see any reference to robot_namespace_ at all.